test/Makefile

```diff
@@ -1,3 +1,4 @@
+# CC=gcc
```

Comment:
Please remove. To build w/ GCC just use `make CC=gcc`.

Reply:
As mentioned bellow, this should not be here, I'll remove it